src/rsz/src/Resizer.tcl

```diff
@@ -612,7 +606,8 @@ proc check_corner_wire_caps {} {
   set have_rc 1
   foreach corner [sta::corners] {
     if { [rsz::wire_signal_capacitance $corner] == 0.0 } {
-      utl::warn RSZ 14 "wire capacitance for corner [$corner name] is zero. Use the set_wire_rc command to set wire resistance and capacitance."
+      utl::warn RSZ 14 "wire capacitance for corner [$corner name] is zero.\
+        Use the set_wire_rc command to set wire resistance and capacitance."
```

Comment:
Won't this introduce extra whitespace into the message?

Reply:
The `\` character will actually replace the newline and all leading whitespace on the next line with just a single space, so it should be the same as before. As an example you can run in `tclsh`, this should just print `Hello World`: 

```tcl
puts "Hello\
  World"
```

Happy to tweak this if you have a different preference on how to do it, though!